### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by rehanmohiuddin**
- Added Desktop Navigation
- Added Mobile Navigation
- Added Hamburger Click Function Which opens Mobile Nav Menu

### CODE REVIEW

File: Components/Navigation/index.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
+.kash-desktop-header {
+  background-color: var(--white);
+  padding: calc(1 * var(--spacing-unit));
```
Comment (shivanigangadharan) [inline Components/Navigation/index.css:3]: I like the use of calc function here. 👍🏼

File: Components/Navigation/index.html [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+      name="viewport"
+      content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0"
+    />
+
+    <link rel="stylesheet" href="../../index.css" />
+
+    <title>Navigation</title>
+  </head>
+  <body>
+    <div id="kash-container" class="kash-container-root">
```
Comment (shivanigangadharan) [inline Components/Navigation/index.html:17]: I didn't understand the need of the word 'kash' here, just a suggestion to use short meaningful id/class names like 'container' or 'container-root' throughout the code.

**Review by shivanigangadharan — APPROVED** [review]
Overall the code looks good, well structured. Do also add either a deploy preview or a screenshot/gif for better understanding of the component.

File: Components/Navigation/index.html [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+        </div>
+      </main>
+    </div>
+    <script>
+      const hamburgerMenuBtn = document.getElementById("nav-btn");
+      hamburgerMenuBtn.onclick = () => {
+        const navBasicDisplay =
+          document.getElementById("nav-mob").style.display;
+        document.getElementById("nav-mob").style.display =
+          navBasicDisplay === "block" ? "none" : "block";
```
Comment (nerdyspook) [inline Components/Navigation/index.html:146]: You can move this block of code to a js file to avoid using js code in HTML.

**Review by nerdyspook — CHANGES_REQUESTED** [review]

File: Components/Navigation/index.html (hunk 2 above)
Comment (rehanmohiuddin) [reply Components/Navigation/index.html:17]: kash is the short name of library here so if it used in any other app that do not overlap the names so i used this i discussed with my team leader he said it's ok to use and should be meaning ful

File: Components/Navigation/index.html (hunk 3 above)
Comment (rehanmohiuddin) [reply Components/Navigation/index.html:146]: short code that's why i have written here

**Pull request merged**